sphinx_gallery/gen_rst.py

```diff
@@ -871,15 +888,20 @@ def rst_blocks(script_blocks, output_blocks, file_conf, gallery_conf):
 
             code_rst = codestr2rst(bcontent, lang=gallery_conf['lang'],
                                    lineno=lineno) + '\n'
+            example_rst += ".. container:: sphx-glr-code\n"
             if is_example_notebook_like:
-                example_rst += code_rst
-                example_rst += code_output
+                example_rst += indent(code_rst, ' '*4)
```

Comment:
this is minor, but instead of doing `' '* 8`, I like to define a tab variable and then use multiples of that. e.g.

```
tab = '    '
indent(code_rst, tab*2)
```

Reduces one cognitive step of dividing the number by the number of spaces per tab :-)